packages/integrations/deno/README.md

```diff
@@ -1,3 +1,3 @@
 # @astrojs/deno 🦖
 
-This adapter is no longer maintained by Astro. Please see [the new repository for the Deno adapter](https://github.com/withastro/netlify-adapter) which is now maintained by the Deno organization.
+This adapter is no longer maintained by Astro. Please see [the new repository for the Deno adapter](https://github.com/denoland/deno-astro-adapter) which is now maintained by the Deno organization.
```

Comment:
@lilnasy this fix already landed on main :)

Reply:
thanks for letting me know, ill rebase